pkg/test/resourcefixture/testdata/basic/gkehub/v1beta1/gkehubfeaturemembership/basiccsaugkehubfeaturemembership/_http.log

```diff
@@ -0,0 +1,2206 @@
+    "endTime": "2024-04-01T12:34:56.123456Z",
+    "target": "projects/example-project-01/locations/global/features/configmanagement"
+  },
+  "name": "projects/example-project-01/locations/global/features/configmanagement/operations/${operationID}",
+  "response": {
+    "@type": "type.googleapis.com/google.cloud.gkehub.v1beta.Feature",
+    "labels": {
+      "cnrm-test": "true",
+      "managed-by-cnrm": "true"
+    },
+    "membershipSpecs": {
+      "projects/example-project-01/locations/us-south1/memberships/gkehubmembership-basic-csau-${uniqueId}": {
+        "configmanagement": {
+          "configSync": {
+            "sourceFormat": "unstructured"
+          },
+          "version": "1.18.1"
+        }
+      }
+    },
+    "name": "projects/example-project-01/locations/global/features/configmanagement",
+    "resourceState": {
+      "state": "ACTIVE"
+    },
+    "updateTime": "2024-04-01T12:34:56.123456Z"
+  }
+}
+
+---
+
+GET https://gkehub.googleapis.com/v1beta/projects/example-project-01/locations/global/features/configmanagement?alt=json&prettyPrint=false
+User-Agent: kcc/controller-manager
+
+200 OK
+Cache-Control: private
+Content-Type: application/json; charset=UTF-8
+Server: ESF
+Vary: Origin
+Vary: X-Origin
+Vary: Referer
+X-Content-Type-Options: nosniff
+X-Frame-Options: SAMEORIGIN
+X-Xss-Protection: 0
+
+{
+  "labels": {
+    "cnrm-test": "true",
+    "managed-by-cnrm": "true"
+  },
+  "membershipSpecs": {
+    "projects/example-project-01/locations/us-south1/memberships/gkehubmembership-basic-csau-${uniqueId}": {
+      "configmanagement": {
+        "configSync": {
+          "sourceFormat": "unstructured"
+        },
+        "version": "1.18.1"
+      }
+    }
+  },
+  "name": "projects/example-project-01/locations/global/features/configmanagement",
+  "updateTime": "2024-04-01T12:34:56.123456Z"
+}
+
+---
+
+PATCH https://gkehub.googleapis.com/v1beta/projects/example-project-01/locations/global/features/configmanagement?alt=json&prettyPrint=false&updateMask=membershipSpecs
+Content-Type: application/json
+User-Agent: kcc/controller-manager
+
+{
+  "labels": {
+    "cnrm-test": "true",
+    "managed-by-cnrm": "true"
+  },
+  "membershipSpecs": {
+    "projects/example-project-01/locations/us-south1/memberships/gkehubmembership-basic-csau-${uniqueId}": {
+      "configmanagement": {
+        "configSync": {
+          "git": {
+            "gcpServiceAccountEmail": "sa-${uniqueId}@example-project-01.iam.gserviceaccount.com",
+            "httpsProxy": "https://github.com",
+            "policyDir": "config-connector",
+            "secretType": "none",
+            "syncBranch": "master",
+            "syncRepo": "https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit",
+            "syncRev": "HEAD",
+            "syncWaitSecs": "20"
+          },
+          "sourceFormat": "hierarchy"
+        },
+        "management": "MANAGEMENT_AUTOMATIC",
+        "version": "1.18.1"
```

Comment:
If I understand it correctly, "management" and "version" are conflicting fields. Could you help me understand why do they both exist in the PATCH request?